### DISCUSSION TITLE
Need assistance in choosing git enterprise subscription

### DISCUSSION THREAD

**Discussion by horizon00xx** [community/community#114479] [Enterprise]
Hello,

We are planning to shift our codebase from local system to VCS, for same we want better understanding of github, based on our requirement. 
Can someone elaborate us which parameters to take in account before shifting codebase and all.

**Comment by AidenJamesYt** [comment]
I would go through githubs documentation :)

**Reply by ethanwater** [reply]
Agreed.

**Comment by ghostinhershell** [comment]
Thanks for posting in the GitHub Community, @horizon00xx!

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the New to GitHub category is solely related to conversations around getting started on using GitHub and the community for beginners of GitHub, Community on Discussions, and those new to different areas of the product alike! This question should be in the Enterprise category. I've gone ahead and moved it for you. Good luck!
